### DISCUSSION TITLE
Not sure where else to post this - two IOTM suggestions.

### DISCUSSION THREAD

**Discussion by danbeardan** [loathers/garbage-collector#1718] [Ideas]
Hey I noticed that my Garbo does not use my Genie Bottle or use my Tome of Smithing. Could we add. "wish for more wishes" or whatever is the most valuable use for a pocket wish to the Genie Bottle?

**Comment by pstalcup** [comment]
This can be configured under mafia's breakfast settings, which we generally do not replicate

**Reply by danbeardan** [reply]
Thanks for the heads up.
